tooling/autodispose-rx-error-prone-checker/gradle.properties

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (C) 2017. Uber Technologies
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+POM_NAME=AutoDispose Rx Error Prone Checker (RxLifecycle Interop)
+POM_ARTIFACT_ID=autodispose-rx-error-prone-checker
```

Comment:
remove the `-rx` from this artifact ID and the project name, it's redundant

Reply:
https://github.com/uber/AutoDispose/pull/156/commits/5cef2be0f4cd0d9e083c739438e96ef13b4394be